__tests__/api.test.js

```diff
@@ -35,3 +35,25 @@ describe("/api/topics", () => {
         });
     });
 });
+
+describe("/api", () => {
+    describe("GET", () => {
+        test("200: status code of 200 is sent", () => {
+            return request(app).get("/api").expect(200);
+        });
+        test("200: Responds with the body of endpoints.json", () => {
+            return request(app)
+                .get("/api")
+                .expect(200)
+                .then(({ body: { endpoints } }) => {
```

Comment:
Nice, it would be best to require in the endpoints file then just compare them to check it looks exactly the same.